### ISSUE TITLE
Meek rule sometimes crashes when handling equal preferences

### ISSUE THREAD

**Issue by brainwad** [jklundell/droop#5]
When testing with an election where all voters voted only with tied first preferences (attached), I got an IndexError here: https://github.com/jklundell/droop/blob/556746c7474228210c64db12b0504775879da759/droop/rules/meek.py#L261

I worked around it by wrapping it in try/catch and just doing `cids = []` if an exception happened; but I'm not totally confident that's actually correct.

Stacktrace:

```
Traceback (most recent call last):
  File "Droop.py", line 172, in <module>
    report = main(eoptions)
  File "Droop.py", line 96, in main
    countElection(E, reps)
  File "Droop.py", line 87, in countElection
    E.count()
  File "/usr/local/google/home/smro/droop/droop/election.py", line 133, in count
    self.rule.count()   ### count the election ###
  File "/usr/local/google/home/smro/droop/droop/rules/meek.py", line 383, in count
    iterationStatus, batch = iterate()
  File "/usr/local/google/home/smro/droop/droop/rules/meek.py", line 290, in iterate
    distributeVotes()
  File "/usr/local/google/home/smro/droop/droop/rules/meek.py", line 275, in distributeVotes
    dist(0, V1)
  File "/usr/local/google/home/smro/droop/droop/rules/meek.py", line 273, in dist
    dist(i+1, weight)
  File "/usr/local/google/home/smro/droop/droop/rules/meek.py", line 264, in dist
    raise e
IndexError: tuple index out of range
```

BLT input: [test.txt](https://github.com/jklundell/droop/files/4593379/test.txt)
repro command: `python Droop.py meek fixed precision=9 omega=6 defeat_batch=none test.txt`

**Comment by jklundell** [comment]
Thanks. Offhand, that sounds like a reasonable workaround. I'll try to take a closer look, and ideally add a test case (thanks for that, too). TBH I haven't looked at this code for a long time (obviously I guess); glad to see someone using it.

**Comment by jklundell** [comment]
Fixed in 0.13. Turns out it was an off-by-one error, iterating over the ballot items once too many times. (The try/except fix would have been a valid workaround, btw.)

Thanks for the report!
